packages/daemons/block_generator.go

```diff
@@ -36,6 +37,9 @@ import (
 // BlockGenerator is daemon that generates blocks
 func BlockGenerator(ctx context.Context, d *daemon) error {
 	d.sleepTime = time.Second
+	if service.NodePaused.IsSet() {
```

Comment:
I doesn't sure that's a good idea, because - are you sure that that flag wouldn't be setted in the middle of block generator function?. I think it would be better to use lock in NodeActualizer, which doen't allow other daemons enter this critical section until it doesn't clear how actual is that node